### PR TITLE
feat: Runner.open to start a one off execution

### PR DESCRIPTION
**Pull request by willemneal**
Unlike the `run` method, this method is immediately executed and no cleanup is performed.  Furthermore, it requires a `rootAccount`.

By default it uses testnet as the network to connect to and the home directory for the keystore.

### CODE REVIEW

**Comment by chadoh** [comment]
Looks like there are conflicts that need to be resolved

File: __tests__/00.using-open.ava.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+ *
+ * Note that the same tests will be run on both a local sandbox environment and
+ * on testnet by using the `test:sandbox` and `test:testnet` scripts in
+ * package.json.
+ */
+import {ava as test, Runner} from 'near-runner-ava';
+
+test('using runner', async t => {
+  await Runner.open({rootAccount: 'meta'}, async ({root}) => {
+    t.is(root.accountId, 'meta');
```
Comment (chadoh) [inline __tests__/00.using-open.ava.ts:14]: I think this needs more explanation. What is the `meta` account? Why is it called that? Why am I allowed to make calls as this account when using `near-runner`? 

I also don't think we should make this test number `00`. I think most people will be coming to near-runner / near-workspaces for testing purposes, at least at the beginning, so we should lead with that.

For this demonstration of `Runner.open`, I think we should show doing something a little more interesting, to try to help people understand what sort of cool scripting they might be able to do. Maybe we could create an account on testnet, or make calls to `wrap.testnet`, and add a comment about how we could hypothetically now exchange it for a different token on testnet's Ref Finance.

File: packages/js/src/runner.ts [hunk 2]
```diff
@@ -92,19 +93,29 @@ export class Runner {
...
-        throw new Error(
-          `environment variable NEAR_RUNNER_NETWORK=${network} invalid; `
-          + 'use \'testnet\' or \'sandbox\' (the default)',
-        );
-    }
+    return getNetworkFromEnv();
+  }
+
+  /**
+   *
```
Comment (chadoh) [inline packages/js/src/runner.ts:100]:
```suggestion
```

File: packages/js/src/utils.ts [hunk 3]
```diff
@@ -91,3 +94,29 @@ export function hashContract(contract: string | Buffer): string {
 }
 
 export const EMPTY_CONTRACT_HASH = '11111111111111111111111111111111';
+
+/**
+ *
+ * @returns network to connect to. Default 'sandbox'
+ */
+export function getNetworkFromEnv(): 'sandbox' | 'testnet' {
```
Comment (chadoh) [inline packages/js/src/utils.ts:102]: Should we add `mainnet` now?

**Review by chadoh — APPROVED** [review]
Looks pretty good; a couple quick notes

File: packages/js/src/utils.ts (hunk 3 above)
Comment (willemneal) [reply packages/js/src/utils.ts:102]: Sure!

**Review by chadoh — CHANGES_REQUESTED** [review]
Looks pretty good; a couple quick notes though

File: packages/js/src/utils.ts (hunk 3 above)
Comment (willemneal) [reply packages/js/src/utils.ts:102]: In retrospect I think it shouldn't be added to this PR as it involves more internal efforts.

**Review by chadoh — APPROVED** [review]

**Pull request merged**